display/renderbasic/src/main/java/org/openscience/cdk/renderer/generators/standard/StandardSgroupGenerator.java

```diff
@@ -487,6 +487,19 @@ private IRenderingElement generateMixtureSgroup(Sgroup sgroup) {
         }
     }
 
+    private static boolean isDigit(char c) {
+        return c >= '0' && c <= '9';
+    }
```

Comment:
Is this faster than Character.isDigit() ?

(aka it seems there are several classes now with a method like this... what is the reason?)